builtins/amp-pixel.js

```diff
@@ -69,9 +70,29 @@ export class AmpPixel extends BaseElement {
           .expandAsync(this.assertSource_(src))
           .then(src => {
             const image = new Image();
-            image.src = src;
-            dev().info(TAG, 'pixel triggered: ', src);
-            return image;
+            const referrerPolicy = this.element.getAttribute('referrerpolicy');
+            // if "referrerPolicy" is not supported, use iframe wrapper
+            // to scrub the referrer.
+            if (image.referrerPolicy === undefined &&
+                referrerPolicy == 'no-referrer') {
```

Comment:
This is the only value we support. That should be checked in the validator and be a user assert here. If the value is empty we can return early.